### PR TITLE
Separate project types from main script 

### LINKED ISSUE #12: Separate project types into their own python module
Either by moving the dictionary and functions together to the new module, or by just moving the functions to the new module

### PR DESCRIPTION
**Pull request by jarodburchill**
Closes #12

### CODE REVIEW

**Comment by jarodburchill** [comment]
@ajnieset @Rexogamer @MisterWhopper Take a look through these changes if you can please!

File: automate_project.py [hunk 1]
```diff
@@ -96,7 +92,6 @@ def CreateGitHubRepo():
 
 
 # loops until there is a valid project name
-print(localPath + "\\" + projectName)
 while os.path.isdir(localPath + "\\" + projectName):
     print(bcolors.WARNING +
```
Comment (Rexogamer) [inline automate_project.py]: Can 96 and 97 be on 1 line?

File: automate_project.py [hunk 2]
```diff
@@ -96,7 +92,6 @@ def CreateGitHubRepo():
...
           "Project name already exists; please try again." + bcolors.ENDC)
```
Comment (Rexogamer) [inline automate_project.py:97]: Random thing: 96/97 could be one line

Comment (Rexogamer) [reply automate_project.py:97]: other than that I can’t see any issues

**Review by ajnieset — APPROVED** [review]

**Review by Rexogamer — APPROVED** [review]

**Pull request merged**